### PR TITLE
One-off (Stripe) acquisition tracking

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

To add acquisition tracking to one-off payments made via stripe.

[**Trello Card**](https://trello.com/b/TjUElW0B/simple-and-coherent-product-supporting-the-guardian)

## Changes

* Adding extra fields to one-off payments.
* Adding user id to the autofill call.

## Screenshot

Request example
picture 413

## Related PR

https://github.com/guardian/contributions-frontend/pull/359

### CODE REVIEW

File: assets/helpers/user/userActions.js [hunk 1]
```diff
@@ -17,6 +18,10 @@ export type Action =
 
 // ----- Actions Creators ----- //
 
+export function setId(id: string): Action {
```
Comment (Ap0c) [inline assets/helpers/user/userActions.js]: `setUserId`?

**Comment by svillafe** [comment]
@Mullefa That was generated by another bug which is not related to this PR. That bug is fix here: https://github.com/guardian/support-frontend/pull/271

**Comment by Mullefa** [comment]
@svillafe ok, cool

**Comment by Mullefa** [comment]
@svillafe and me have tested this on CODE in conjunction with contributions frontend running on CODE and it's all good. I'll let you know when [#359](https://github.com/guardian/contributions-frontend/pull/359) is seen in PROD, then we can merge this pull request.

**Pull request merged**